ModelVale/ModelVale/CustomClasses/CustomObjects/DatabaseObjects/ModelData.m

```diff
@@ -41,27 +41,29 @@ - (instancetype)initWithDictionary:(NSDictionary *)dict {
 }
 
 //MARK: Firebase
-+ (void) fetchFromReference: (FIRStorage*)storage docRef: (FIRDocumentReference*)docRef vc: (UIViewController*)vc completion:(void(^)(ModelData*))completion {
++ (void) fetchFromReference: (FIRDocumentReference*)docRef storage: (FIRStorage*)storage vc: (UIViewController*)vc completion:(void(^)(ModelData*))completion {
     [docRef getDocumentWithCompletion:^(FIRDocumentSnapshot * _Nullable snapshot, NSError * _Nullable error) {
         if(error != nil) {
             [vc presentError:@"Failed to fetch ModelData" message:error.localizedDescription error:error];
         }
         else {
             ModelData* d = [ModelData initWithImage:nil label:snapshot.data[@"label"] imagePath:snapshot.data[@"imagePath"]];
```

Comment:
try not to be more descriptive with your variables names so anyone reading your code could know just by looking at the name what is that variable for, and avoid using single characters for naming variables